autobox.js

```diff
@@ -100,7 +101,7 @@ function unboxWithCache (id) {
     const cached = cache.get(msg.key)
 
     if (cached === false && !readKey) return msg
-    else if (cached) return cached
+    else if (cached) return cloneDeep(cached)
```

Comment:
ah yes I see, I think this is quite the right place to put this... on the exit ensure _no-one_ can mutate the cache

Reply:
:heart: